### ISSUE TITLE
Improve docker-compose configuration detection

### ISSUE THREAD

**Issue by tadaskay** [avast/docker-compose-gradle-plugin#52]
I usually use this plugin in a multi-module project where `docker-compose.yml` is located in the root project directory. ComposeUp and ComposeDown tasks work fine (they depend on `docker-compose` behavior), but healthcheck will not take place because serviceInfos will not be resolved (it is looking for `docker-compose.yml` in the current directory, not in the `rootProject.dir`):

https://github.com/avast/docker-compose-gradle-plugin/blob/c4d794c72147e4a13136f15aa6d2387a91c878a2/src/main/groovy/com/avast/gradle/dockercompose/tasks/ComposeUp.groovy#L70

```
String[] composeFiles = extension.useComposeFiles.empty ? ['docker-compose.yml', 'docker-compose.override.yml'] : extension.useComposeFiles
composeFiles
            .findAll { project.file(it).exists() }
<...>
```

I had to debug the plugin to find out that I was missing 
```
dockerCompose {
    useComposeFiles = ['../docker-compose.yml']
}
```

It would be nice to have a hint that docker-compose file cannot be located in the expected location or even better, improve docker-compose.yml detection, mimicking docker-compose:

https://docs.docker.com/compose/reference/overview/

> The -f flag is optional. If you don’t provide this flag on the command line, Compose traverses the working directory and its parent directories looking for a docker-compose.yml and a docker-compose.override.yml file.

**Comment by augi** [comment]
The logging of non-existing files is a must - I will do it ASAP.

I'm not sure if I understand why you had to use this:
```gradle
dockerCompose {
    useComposeFiles = ['../docker-compose.yml']
}
```
Did you applied the plugin on the child-project?

**Comment by tadaskay** [comment]
Yes. An example of such project could be acceptance tests (that are executed against the whole stack of containers).

**Comment by augi** [comment]
Released as `0.3.17`. If you use Docker Compose 1.6.0 or newer then a new way how to list all services is used (`docker-compose config --services`).

**Comment by tadaskay** [comment]
I can verify it works fine now. Thanks for the quick response 🥇
